drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -134,9 +134,10 @@ def get_form_definition(enrollment_id:, assessment_role:)
     field :pick_list, [Types::Forms::PickListOption], 'Get list of options for pick list', null: false do
       argument :pick_list_type, Types::Forms::Enums::PickListType, required: true
       argument :project_id, ID, required: false
+      argument :inventory_id, ID, required: false
     end
-    def pick_list(pick_list_type:, project_id: nil)
-      Types::Forms::PickListOption.options_for_type(pick_list_type, user: current_user, project_id: project_id)
+    def pick_list(pick_list_type:, project_id: nil, inventory_id: nil)
+      Types::Forms::PickListOption.options_for_type(pick_list_type, user: current_user, project_id: project_id, inventory_id: inventory_id)
```

Comment:
Same here, if project_id and inventory_id are never both set, seems like a more generic parent_id or relation_id would be convenient